app.py

```diff
@@ -34,35 +37,40 @@ def __init__(self, session_state=st.session_state):
                     st.session_state.app_initialized = True
         # Answerable and Unanswerable questions
         self.answerable_questions = {
-            "How can I contact ITS?",
-            "How can I connect to the campus Wi-Fi?",
-            "Who are the Co-Chairs for the 2024/2025 Committee?",
-            "Where are all the printers located?",
-            "What are the CoyoteLabs virtual computer labs?",
-            "Is Adobe Creative Cloud available as student software?",
-            "Does CSUSB have accessible technology?",
-            "How do I enable multi-factor authentication?",
-            "What are Coyote OneCard benefits?",
-            "Why can't I get access for wireless prints through my phone?",
+            "How can I contact ITS",
+            "How can I connect to the campus WiFi",
+            "What are the available free software for a student",
+            "Where are all the printers located",
+            "What are the CoyoteLabs virtual computer labs",
+            "Is Adobe Creative Cloud available as student software",
+            "What is information security awareness",
+            "How do I enable multi-factor authentication",
+            "What are Coyote OneCard benefits",
+            "What if i lost my campus laptop charger",
         }
         self.answerable_questions = {q.lower() for q in self.answerable_questions}
```

Comment:
Can you change this to use the `remove_special_characters` instead of `.lower()`?